entities/global.ent

```diff
@@ -651,11 +651,11 @@
 <!ENTITY spec.pdf.fdf   'http://www.adobe.com/devnet/acrobat/fdftoolkit.html'>
 
 <!-- The three special language constants -->
-<!ENTITY true  '<constant xmlns="http://docbook.org/ns/docbook">TRUE</constant>'>
-<!ENTITY false '<constant xmlns="http://docbook.org/ns/docbook">FALSE</constant>'>
-<!ENTITY null  '<constant xmlns="http://docbook.org/ns/docbook">NULL</constant>'>
-<!-- use <type>NULL</type> if you're talking about the _type_ NULL,
-     use &null; if it's about the _value_ NULL
+<!ENTITY true  '<constant xmlns="http://docbook.org/ns/docbook">true</constant>'>
+<!ENTITY false '<constant xmlns="http://docbook.org/ns/docbook">false</constant>'>
+<!ENTITY null  '<constant xmlns="http://docbook.org/ns/docbook">null</constant>'>
+<!-- use <type>null</type> if you're talking about the _type_ null,
```

Comment:
In the sense of this comment, I believe I'll have to modify the methodsynopsis generator not to generate `&null` as part of an union type

Reply:
Well, IMO `<type>null</type>` is good enough.